pkg/splunk/enterprise/names.go

```diff
@@ -60,13 +60,13 @@ const (
 	commandMerger = " && "
 
 	// command for the init container with DFC
-	commandForDfc = "cp -r /opt/jdk /mnt && cp -r /opt/spark /mnt"
+	//commandForDfc = "cp -r /opt/jdk /mnt"
 
 	// command for init container on a standalone without DFC
 	commandForStandaloneSmartstore = "mkdir -p /opt/splk/etc/apps/splunk-operator/local && ln -sfn  /mnt/splunk-operator/local/indexes.conf /opt/splk/etc/apps/splunk-operator/local/indexes.conf && ln -sfn  /mnt/splunk-operator/local/server.conf /opt/splk/etc/apps/splunk-operator/local/server.conf"
 
 	// command for init container with Smartstore enabled and DFC enabled
-	commandForDfcAndSmartstore = commandForDfc + commandMerger + commandForStandaloneSmartstore
+	//commandForDfcAndSmartstore = commandForDfc + commandMerger + commandForStandaloneSmartstore
```

Comment:
Are we commenting here vs removing?

Reply:
Good catch @akondur. We'll remove it on next commit.